app/services/action_handler.rb

```diff
@@ -0,0 +1,23 @@
+class ActionHandler < BaseService
+  def initialize(payload:, event:, event_type:)
+    @payload = payload
+    @event = event
+    @event_type = event_type
```

Comment:
Does it make sense to rename this to something like `entity`? Because in fact it holds the actual entity we are going to update (PullRequest, Review, etc), right?

Reply:
Yes, you are right! Thanks!